de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/session/SessionUtils.java

```diff
@@ -24,31 +22,27 @@
 // separated
 @Deprecated
 public class SessionUtils {
-  private static volatile ISarosSession currentSarosSession;
-
-  @Inject private static ISarosSessionManager sarosSessionManager;
 
-  static {
-    SarosPluginContext.initComponent(new SessionUtils());
+  @SuppressWarnings("FieldCanBeLocal")
+  private ISessionLifecycleListener lifecycleListener =
+      new ISessionLifecycleListener() {
+        @Override
+        public void sessionStarted(ISarosSession session) {
 
-    sarosSessionManager.addSessionLifecycleListener(
-        new ISessionLifecycleListener() {
-          @Override
-          public void sessionStarted(ISarosSession session) {
+          currentSarosSession = session;
+        }
 
-            currentSarosSession = session;
-          }
+        @Override
+        public void sessionEnded(ISarosSession session, SessionEndReason reason) {
 
-          @Override
-          public void sessionEnded(ISarosSession session, SessionEndReason reason) {
+          currentSarosSession = null;
+        }
+      };
 
-            currentSarosSession = null;
-          }
-        });
-  }
+  private static volatile ISarosSession currentSarosSession;
 
-  private SessionUtils() {
-    // NOP
+  public SessionUtils(ISarosSessionManager sarosSessionManager) {
```

Comment:
Maybe you should add a comment that this must not be called.

Reply:
Done.